src/components/multiple-post-select.js

```diff
@@ -12,7 +12,7 @@ export const MultiplePostSelect = ( { attributes, setAttributes } ) => {
 
 	const postTypes = useSelect( ( select ) =>
 		select( coreStore )
-			.getPostTypes()
+			.getPostTypes({ per_page: -1 })
```

Comment:
I'd prefer that we have a set value here. `-1` queries are dangerous. I realize that there's not a big chance of this becoming an issue but can we set it to a number like 50 instead?

Reply:
I can test it out. I think it may be related to a bug since this post type query in the data store is not supposed to be inherently paginated. Even the core query block does it like this 

https://github.com/WordPress/gutenberg/blob/d6f8ce2b8d4823ca0c97316b29dcc10356ff9817/packages/block-library/src/query/utils.js#L100C67-L100C70

Reply:
Hmm. Let me know how the tests go.